.eslintignore

```diff
@@ -1,11 +1,10 @@
-localforage.js
+localstorage.js
```

Comment:
First: localforage was actually correct. What makes you think it should be renamed? Second: that file was actually deleted. So this rule can go away. Please remove it.